guides/common/modules/proc_creating-a-content-view.adoc

```diff
@@ -10,23 +10,34 @@ While you can stipulate whether you want to resolve any package dependencies on
 For more information, see xref:Resolving_Package_Dependencies_{context}[].
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create New View*.
+. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create content view*.
 . In the *Name* field, enter a name for the view.
 {Project} automatically completes the *Label* field from the name you enter.
 . In the *Description* field, enter a description of the view.
-. Optional: if you want to solve dependencies automatically every time you publish this Content View, select the *Solve Dependencies* checkbox.
+. In the *Type* field, select a *Content view* or a *Composite content view*.
+. Optional: If you want to solve dependencies automatically every time you publish this Content View, select the *Solve dependencies* check box.
 Dependency solving slows the publishing time and might ignore any Content View filters you use.
 This can also cause errors when resolving dependencies for errata.
-. Click *Save* to create the Content View.
-. In the *Repository Selection* area, select the repositories that you want to add to your Content View, then click *Add Repositories*.
-. Click *Publish New Version* and in the *Description* field, enter information about the version to log changes.
-. Click *Save*.
-. Optional: to force metadata regeneration on Yum repositories, from the *Actions* list for your Content View versions, select *Regenerate Repository Metadata*.
+. Optional: If you want to designate this Content View for importing from an upstream server, select the *Import only* check box.
+Import-only content views cannot be published directly.
+. Click *Create content view*.
+
+.Content View Steps
+. Click *Create content view* to create the Content View.
```

Comment:
```suggestion
```

IMHO not necessary.

Reply:
> IMHO not necessary.

I’m not happy with it either, but i wanted to merge this first, then open a PR against master to address this and some other issues related in the chapter (this particular line should be changed in master, 3.3, and 3.4). Or I can just cherry-pick it into more versions. WDYT?

Reply:
Good point. Then a follow-up PR is the way to go.